File: CodeExamples/PlayWave.cs
//This example just plays a wave stored in a wave bank, fetching it with its ID
using System;
using System.Reflection;

class Program
{
    static void Main()
    {
        try
        {
            var asm = Assembly.LoadFrom(AppDomain.CurrentDomain.BaseDirectory + "WAVEBANK.dll");
            var type = asm.GetType("WAVEBANK.WBKPlayerLib.Player");
            var player = Activator.CreateInstance(type);
            type.GetMethod("LoadWBK").Invoke(player, new object[] { "BANK.wbk" }); //Replace this with the path to the bank
            type.GetMethod("PlayWave").Invoke(player, new object[] { "001" });
        }
        catch { }

        Console.ReadKey();
    }
}

File: CodeExamples/PlayWave.vb
'This example just plays a wave from a wave bank, fetching it by its ID
Imports System
Imports System.Reflection

Module Program
    Sub Main()
        Try
            Dim asm = Assembly.LoadFrom(AppDomain.CurrentDomain.BaseDirectory & "WAVEBANK.dll")
            Dim type = asm.GetType("WAVEBANK.WBKPlayerLib.Player")
            Dim player = Activator.CreateInstance(type)
            type.GetMethod("LoadWBK").Invoke(player, New Object() {"BANK.wbk"}) 'Replace with the path to the bank
            type.GetMethod("PlayWave").Invoke(player, New Object() {"001"})
        Catch
        End Try

        Console.ReadKey()
    End Sub
End Module
